### PR TITLE
Move the pipeline to ADO to allow runs from fork

### PR DESCRIPTION
**Pull request by manvkaur**
Move the PR and CI build to ADO to allow runs from forks with a comment from maintainer

Possible conflict with #30

### CODE REVIEW

File: build/build-pr-ci.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+steps:
+  - task: UseDotNet@2
+    displayName: 'Install .NET Core SDK 6 LTS'
+    inputs:
+      version: '6.x'
+
+  - task: UseDotNet@2
+    displayName: 'Install .NET Core SDK 7 STS'
+    inputs:
+      version: '7.x'
```
Comment (cgillum) [inline build/build-pr-ci.yml:33]: This should be 8.x. Generally speaking, we should avoid depending on STS releases when possible.

**Review by cgillum — COMMENTED** [review]
LGTM. I have one suggestion below. Also, I think we should wait to merge this after merging @aishwaryabh's PR.

File: build/build-pr-ci.yml (hunk 1 above)
Comment (manvkaur) [reply build/build-pr-ci.yml]: Yes, it should have been 8, updated.

**Review by cgillum — APPROVED** [review]

**Comment by manvkaur** [comment]
Addressing https://github.com/Azure/azure-functions-pyfx-planning/issues/195

**Pull request merged**